src/components/pages/Clubs/ViewClubs.js

```diff
@@ -73,7 +72,7 @@ function ViewClubs(props) {
             title={'Clubs'}
             subTitle={`Sorted by clubs`}
           />
-          <ImportClubs fetchClubs={fetchClubs} />
+          <ImportClubs fetchClubs={() => getClubs('authState', context)} />
```

Comment:
Good job keeping the work on this feature within scope and not digging into legacy code